tests/slr_integration/Frontend/Timer_Test.php

```diff
@@ -69,6 +69,31 @@ public function reset_cookie(): void {
 		unset( $_COOKIE[ Session::COOKIE_NAME ] );
 	}
 
+	public function asset_data_provider() {
+		$assets = [
+			'tec-tickets-seating-session'       => '/build/Seating/frontend/session.js',
+			'tec-tickets-seating-session-style' => '/build/Seating/frontend/session.css',
+		];
+
+		foreach ( $assets as $slug => $path ) {
+			yield $slug => [ $slug, $path ];
```

Comment:
If you use a Generator and yield from an array, you might as well return an array.
PHPUnit will eagerly resolve the Generator anyway to an array anyway.

If you want to use the Generator, then just `yield` twice:

```php
public function asset_data_provider(): Generator {
  yield 'tec-tickets-seating-session' => '/build/Seating/Frontend/session.js`;
  yield 'tec-tickets-seating-session-style' => '/build/Seating/Frontend/session.css`;
}
```

Reply:
I want to pass the slug and the path though. Doing that through a foreach seemed like the easier way.